src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -220,13 +242,52 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                       </Tooltip>
                     </Code>
-
+        
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative/cml_dvc_case/pull/4">
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-dvc/pull/1">
                         <Image src="/img/github/dvc-report.png" alt="GitHub DVC report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-dvc">
+                      <div><span>image: iterativeai/cml:0-dvc2-base1</span></div>
+                      <div><span>pipelines:</span></div>
+                      <div><span>  default:</span></div>
+                      <div><span>    - step:</span></div>
+                      <div><span>        name: Train model</span></div>
+                      <div><span>        script: </span></div>
+                      <Tooltip type="dvc">
+                        <div><span>          - dvc pull data</span></div>
+                      </Tooltip>
+                      <div><span>      </span></div>
+                      <Tooltip type="dependencies">
+                        <div><span>          - pip install -r requirements.txt</span></div>
+                        <div><span>          - dvc repro</span></div>
+                      </Tooltip>
+                      <div><span>      </span></div>
+                      <Tooltip type="reports">
+                        <div><span>          # Compare metrics to main</span></div>
+                        <div><span>          - git fetch --depth=1 origin main:main</span></div>
+                        <div><span>          - dvc metrics diff --show-md main &gt;&gt; report.md</span></div>
+                        <div><span>          # Plot training loss function diff</span></div>
+                        <div><span>          - dvc plots diff </span></div>
+                        <div><span>            --target loss.csv --show-vega main &gt; vega.json</span></div>
+                        <div><span>          - vl2png vega.json &gt; plot.png</span></div>
+                        <div><span>          - echo &#x27;![](./plot.png &quot;Training Loss&quot;)&#x27; &gt;&gt; report.md</span></div>
+                        <div><span>          # Post CML report as a comment in GitLab</span></div>
+                        <div><span>          - cml comment create report.md</span></div>
+                      </Tooltip>
+                    </Code>
+                    <ExampleBox title="CML Report">
+                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-test/cml-example-dvc">
```

Comment:
- [ ] TODO @0x2b3bfa0 

```suggestion
                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-ai/cml-example-dvc/pull-requests/1">
```

Reply:
```suggestion
                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-ai/cml-dvc-case/pull-requests/1">
```